docs/site/Validation-REST-layer.md

```diff
@@ -175,3 +175,76 @@ of 10 digits separated by `-` after the 3rd and 6th digits.
 ```
 
 {% include tip.html content="RegExp can be converted into a string with `.source` to avoid escaping backslashes" %}
+
+## Customize validation errors
+
+Since the error is being caught at the REST layer, the simplest way to customize
+the errors is to customize the
+[sequnce](https://loopback.io/doc/en/lb4/Sequence.html). It exists in all
+LoopBack applications scaffolded by using the `lb4` command and can be found in
+`src/sequence.ts`.
+
+Let's take a closer look at how to customize the error. A few things to note in
+the below code snippet:
+
+1. inject RestBindings.ERROR_WRITER_OPTIONS for logging error
+2. customize error for particular endpoints
+3. create a new error with customized properties
+4. log the error using RestBindings.SequenceActions.LOG_ERROR
+
+{% include code-caption.html content="/src/sequence.ts" %}
+
+```ts
+// 1. inject RestBindings.ERROR_WRITER_OPTIONS for logging error
```

Comment:
Can you please include the line defining the sequence class, to make it more clear that both `constructor` and `handle` belong to the sequence class? This may be also needed to get property syntax highlighting.